addons/payment_stripe/static/src/js/express_checkout_form.js

```diff
@@ -20,53 +20,42 @@ paymentExpressCheckoutForm.include({
             return this._super(...arguments);
         }
 
-        const isShippingInformationRequired = this._isShippingInformationRequired();
         const stripeJS = Stripe(
             providerData.stripePublishableKey, _prepareStripeOptions(providerData)
         );
-        this.stripeExpressCheckoutForm = stripeJS.paymentRequest({
-            country: providerData.countryCode,
-            currency: this.txContext.currencyName,
-            total: {
-                label: this.txContext.merchantName,
-                amount: this.txContext.minorAmount,
-                // Delay the display of the amount until the shipping price is retrieved.
-                pending: isShippingInformationRequired,
+        const paymentRequest = stripeJS.paymentRequest({
+            ...{
+                country: providerData.countryCode,
+                currency: this.txContext.currencyName,
+                requestPayerName: true, // Force fetching the billing address for Apple Pay.
+                requestPayerEmail: true,
+                requestPayerPhone: true,
+                requestShipping: this._isShippingInformationRequired(),
             },
```

Comment:
You don't need to unpack an object if you are creating it right then and there:
```suggestion
            country: providerData.countryCode,
            currency: this.txContext.currencyName,
            requestPayerName: true, // Force fetching the billing address for Apple Pay.
            requestPayerEmail: true,
            requestPayerPhone: true,
            requestShipping: this._isShippingInformationRequired(),
```